index.js

```diff
@@ -52,4 +52,147 @@ const state = {
     }
   ],
   cart: []
-};
\ No newline at end of file
+};
+
+
+function listItems(){
+  for (let i = 0; i < state.items.length; i++) {
+const item = state.items[i];
+const header = document.querySelector("#store")
+const ul = document.querySelector('.item-list')
```

Comment:
define these variables outside of the function incase you plan to use them for other functions too